Maquetado/home.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Lista de Libros</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+<body>
+    <div class="background">
+        <nav class="nav">
+            <img class="nav-logo" src="assets/LogoWolox.png" alt="Logo">
```

Comment:
```suggestion
            <img class="nav-logo" src="assets/LogoWolox.png" alt="Wolox">
```
You can omit the word "Logo" because the crawlers (SEO) assuming that it's a Logo